.depalignrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ignore": {
+    "@babel/core": [
```

Comment:
I just assumed these are intentional for now. Should these match?

Reply:
Good to match things and reduce this when we can I think. Not sure on if they were intentional though, maybe someone else knows.